ingestors/hackage.go

```diff
@@ -12,29 +12,29 @@ import (
 const hackageSchedule = "*/5 * * * *"
 const hackageReleasesUrl = "https://hackage.haskell.org/packages/recent.rss"
 
-type hackage struct {
+type Hackage struct {
```

Comment:
capitalizing these newer types was just for consistency with the other types, and isn't technically necessary.